catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventToXmlElementConverter.java

```diff
@@ -136,16 +137,18 @@ private SaxEventToXmlElementConverter startConstructingElement(
           /*
            * Get the first non-empty prefix that is associated with the URI (the other, non-default prefix)
            */
-          String altNS =
+          Optional<String> altNs =
               namespaceStack
                   .stream()
                   .filter(p -> attUri.equals(p.getUri()) && !(p.getPrefix().isEmpty()))
-                  .findFirst()
-                  .get()
-                  .getPrefix();
-          out.writeNamespace(altNS, attUri);
-          addedNamespaces.put(attUri, altNS);
-          out.writeAttribute(altNS, attUri, atts.getLocalName(i), atts.getValue(i));
+                  .map(NamespaceMapping::getPrefix)
+                  .findFirst();
```

Comment:
✏️  Add '.orElse(null)' to avoid calling isPresent and get on it. Or again, simply add `.ifPresent(altNs -> { your code }).`